policyengine_us/variables/gov/states/nc/ncdhhs/scca/nc_scca_age_group.py

```diff
@@ -0,0 +1,53 @@
+from policyengine_us.model_api import *
+
+
+class AgeGroup(Enum):
+    NOT_QUALIFY = "Not qualify"
+    INFANT = "Infant"
+    TWO_YEAR_OLD = "2 Year olds"
+    THREE_TO_FIVE_YEAR_OLD = "3 - 5 Year olds"
+    SCHOOL_AGE = "School age"
+
+
+class nc_scca_age_group(Variable):
+    value_type = Enum
+    possible_values = AgeGroup
+    default_value = AgeGroup.NOT_QUALIFY
+    entity = Person
+    label = "NC SCCA age group"
+    definition_period = YEAR
+    reference = (
+        "https://docs.google.com/spreadsheets/d/1y7p8qkiOrMAM42rtSwT_ZXeA5tzew4edNkrTXACxf4M/edit?gid=1339413807#gid=1339413807"
+        "https://ncchildcare.ncdhhs.gov/Portals/0/documents/pdf/M/Market_Rates_Centers_Eff_10-1.pdf?ver=9w52alSPhmrmo0N9gGVMEw%3d%3d"
+    )
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.nc.ncdhhs.scca
+        age = person("age", period)
+        disabled = person("is_disabled", period)
+
+        infant_age_limit = p.infant_age_limit  # 2
+        toddler_age_limit = p.toddler_age_limit  # 3
+        preschool_age_lower = p.preschool_age_lower  # 3
+        preschool_age_upper = p.preschool_age_upper  # 6
+        school_age_limit = p.school_age_limit  # 13
+        disabled_age_limit = p.disabled_age_limit  # 18
+
+        return select(
+            [
+                age >= disabled_age_limit,
+                age < infant_age_limit,
+                age < toddler_age_limit,
+                (age >= preschool_age_lower) & (age < preschool_age_upper),
+                (age < school_age_limit)
+                | ((age < disabled_age_limit) & disabled),
```

Comment:
```suggestion
        preschool_age_lower = p.preschool_age_lower  # 3
        preschool_age_upper = p.preschool_age_upper  # 6
        school_age_limit = p.school_age_limit  # 13
        disabled_age_limit = p.disabled_age_limit  # 18

        return select(
            [
                age >= disabled_age_limit,
                age < p.infant_age_limit  # 2,
                age < p.toddler_age_limit  # 3,
                (age >= preschool_age_lower) & (age < preschool_age_upper),
                (age < school_age_limit)
                | ((age < disabled_age_limit) & disabled),
```

Reply:
We can consolidate this - shouldn't need the separate call above - for all conditions

Reply:
please keep the `p.` in when referencing raw parameters to make that clear. not saving us many characters here